tests/run/ext_attr_getter.srctree

```diff
@@ -26,28 +33,55 @@ typedef struct {
     int f2;
 } FooStructNominal;
 
+typedef struct {
+    PyObject_HEAD
+} FooStructOpaque;
+
+
+#define PyFoo_GET0M(a) ((FooStructNominal*)a)->f0
+#define PyFoo_GET1M(a) ((FooStructNominal*)a)->f1
+#define PyFoo_GET2M(a) ((FooStructNominal*)a)->f2
+
+int PyFoo_Get0F(FooStructOpaque *f)
+{
+    return ((FooStructNominal*)f)->f0;
```

Comment:
Shouldn't this use the `#define` macros above?

Reply:
either way ends up compiling to the same thing, adopting the idea macro use.